File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_TKIT_ORDER_DETAILS.ts
export class VM_TKIT_ORDER_DETAILS {
    public CHK_VALUE: any;
    public ORDER_NUMBER: number;
    public ORDER_LINE_NUMBER: number;
    public ORDER_DATE: string;// Date;
    public DELIVER_ITEM_STATUS: string;
    public ITEM_ID: string;
    public REQUEST_QTY?: number;
    public LOCATION_ID: string;
    public ESTIMATED_RETURN_DATE: any;//Date
    public ITEM_DESCR: string;
    public ORDER_COMMENTS: string;
    public IMAGE: string;
    public ITEM_TYPE_INDICATOR: string;
    public CHKDisable: boolean = false;
    public blnErDateTime: boolean = true;
}

File: RSPAClient/Core/AtPar.ViewModel/VM_TKIT_ORDER_DETAILS.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
    public class VM_TKIT_ORDER_DETAILS
    {
        public int CHK_VALUE { get; set; }
        public int ORDER_NUMBER { get; set; }
        public int ORDER_LINE_NUMBER { get; set; }
        public Nullable<System.DateTime> ORDER_DATE { get; set; }
        public string DELIVER_ITEM_STATUS { get; set; }
        public string ITEM_ID { get; set; }
        public Nullable<int> REQUEST_QTY { get; set; }
        public string LOCATION_ID { get; set; }
        public string ESTIMATED_RETURN_DATE { get; set; }//Nullable<System.DateTime>
        public string ITEM_DESCR { get; set; }
        public string ORDER_COMMENTS { get; set; }
        public string IMAGE { get; set; }
        public string ITEM_TYPE_INDICATOR { get; set; }

    }
}
